### ISSUE TITLE
Add directive to support float_cmp

### ISSUE THREAD

**Issue by drdavella** [astropy/pytest-doctestplus#3]

**Comment by pllim** [comment]
Not sure what this means. We already have FLOAT_CMP

**Comment by bsipocz** [comment]
I think this is still relevant as it's about a directive which afaik is still not supported

**Comment by pllim** [comment]
Like... to control a whole block instead of line-by-line?

**Comment by bsipocz** [comment]
Yes. Although I would say having that for remote data is much more important as it's a blocker for astroquery docs
